src/components/study-container.js

```diff
@@ -577,49 +577,23 @@ export function StudyContainer({ view, onChangeTab }) {
 
     const checkStudyIndexation = useCallback(() => {
         setIsStudyIndexationPending(true);
-        fetchStudyIndexationStatus(studyUuid)
-            .then((status) => {
-                switch (status) {
-                    case STUDY_INDEXATION_STATUS.INDEXED: {
-                        dispatch(setStudyIndexationStatus(status));
-                        setIsStudyIndexationPending(false);
-                        break;
-                    }
-                    case STUDY_INDEXATION_STATUS.INDEXING_ONGOING: {
-                        dispatch(setStudyIndexationStatus(status));
-                        break;
-                    }
-                    case STUDY_INDEXATION_STATUS.NOT_INDEXED: {
-                        dispatch(setStudyIndexationStatus(status));
-                        reindexAllStudy(studyUuid)
```

Comment:
remove `fetchStudyIndexationStatus` and `reindexAllStudy` definitions if not used anymore